frontend/src/container/FeedItemWithPreview.jsx

```diff
@@ -356,7 +356,7 @@ export class FeedItemWithPreview extends React.Component {
                   workspaceId={Number(props.workspaceId)}
                   author={commentToShow.author}
                   loggedUser={loggedUser}
-                  created={commentToShow.created_raw || commentToShow.createdRaw || commentToShow.created}
+                  createdDate={commentToShow.created_raw || commentToShow.createdRaw || commentToShow.created}
```

Comment:
creationDate ?

Reply:
Solved in [fd85e03](https://github.com/tracim/tracim/pull/5846/commits/fd85e03f05599f33b4ee4f2e616fd52f78605da7).